src/manifest.js

```diff
@@ -34,7 +34,7 @@ export const parseManifest = ({
   manifestString,
   customTagParsers = [],
   customTagMappers = [],
-  experimentalLLHLS
+  llhls
```

Comment:
The change to the JSDoc isn't enough to default it to `true`. It would need to be changed here as well
```suggestion
  llhls = true
```